src/DotnetWebApiBench.Api/Startup.cs

```diff
@@ -58,7 +59,19 @@ public virtual void ConfigureServices(IServiceCollection services)
                 {
                     fv.RegisterValidatorsFromAssemblyContaining<ProductsController>();
                 });
-            services.AddNorthwindDataAccess(Configuration["ConnectionStrings:Northwind"]);
+            DbTypeEnum dbtype;
+            if (Enum.TryParse(Configuration["db-type"], out dbtype))
```

Comment:
I think checking if TryParse return true will prevent the context to be registered at all if someone either do not provide the database type or the database type is spelled wrongly. In such cases we want to default to SQLite.

Reply:
changes applied